test/ViewExtensionLibraryTests/Setup.cs

```diff
@@ -10,7 +10,7 @@ public class Setup
 {
     private AssemblyHelper assemblyHelper;
 
-    [SetUp]
+    [OneTimeSetUp]
```

Comment:
We need to be careful about this. It could either be one time or could be a setup to be run before each test, in which case, we must leave it as `Setup` attribute. Can you check?

Reply:
All tests passed locally for me but, I will check.